resources/reps-platform.make

```diff
@@ -31,7 +31,7 @@ projects[media_browser_plus][version] = 3.0-beta4
 projects[media_browser_plus][patch][] = "https://www.drupal.org/files/issues/download-files-in-media-basket-does-not-work-2821063-2.patch"
 
 projects[menu_admin_per_menu][subdir] = "contrib"
-projects[menu_admin_per_menu][version] = 1.0
+projects[menu_admin_per_menu][version] = 1.1
 
 projects[multiform][subdir] = "contrib"
 projects[multiform][version] = 1.4
```

Comment:
Needs to be updated to minimum version - 1.6

Reply:
@zarabatana 
I don't think there is a version 1.6 of the module Menu Admin per Menu: https://www.drupal.org/project/menu_admin_per_menu/releases?api_version%5B%5D=103

Reply:
You're right. Last one is 1.1